src/tests.py

```diff
@@ -250,6 +250,14 @@ class CircuitBreakerConfigurationTestCase(object):
     Tests for the CircuitBreaker class.
     """
 
+    def test_default_state(self):
+        """CircuitBreaker: it should get initial state from state_storage.
+        """
+        for state in (STATE_OPEN, STATE_CLOSED, STATE_HALF_OPEN):
+            storage = CircuitMemoryStorage(state)
+            breaker = CircuitBreaker(state_storage=storage)
+            self.assertEqual(breaker._state.name, state)
+
```

Comment:
It'd be good to have a test case to test your `ValueError` exception, too.

Reply:
Oh, and it would also be good to verify that the notify doesn't occur on `CircuitBreaker` instantiation.